### DISCUSSION TITLE
sampler list notation

### DISCUSSION THREAD

**Discussion by chuckkay** [vladmandic/sdnext#1473] [Q&A]
how can i add a personal comment next to each sampler in the sampler drop down list
was just taking a guess and tried editing config.json like this but produced errors, 
        "Euler a",
	"UniPC" # quality fair use 20-30 steps,
        "DPM++ 2M Karras"# quality fair use 20-30 steps,
        "DDIM" # use 10-15 steps,
        "DPM++ 2M SDE Karras" #quality good but slow use 10-15 steps,
        "DPM++ 2S a Karras"
   
also tried moving the comma to after the quotes like this but same error.  
        "Euler a",
	"UniPC", # quality fair use 20-30 steps
        "DPM++ 2M Karras", # quality fair  use 20-30 steps
        "DDIM", # use 10-15 steps
        "DPM++ 2M SDE Karras", #quality good but slow use 10-15 steps
        "DPM++ 2S a Karras"

any suggestions as it would be helpful to remember was works best

**Comment by brknsoul** [comment]
You'd want to follow the notation here;
https://github.com/vladmandic/automatic/wiki/UI-JSON

And they're already being worked in the Settings section.

**Reply by vladmandic** [reply]
yes, but this is reserved for system tooltips/hints.
adding personal annotations for anything doesn't really exist in a concept. and come to think of it, can you name a software that has that?
